### PR TITLE
lp1708073: Fix client help showing a 5.6 link

### PR DESCRIPTION
**Pull request by inikep**

### CODE REVIEW

**Comment by laurynas-biveinis** [comment]
This is https://github.com/percona/percona-server/pull/2017 continued

File: client/mysql.cc [hunk 1]
```diff
@@ -3499,12 +3499,15 @@ com_help(String *buffer MY_ATTRIBUTE((unused)),
 	  return com_server_help(buffer,line,help_arg);
   }
 
-  put_info("\nFor information about Percona products and services, visit:\n"
+  my_snprintf((char*) glob_buffer.ptr(), glob_buffer.alloced_length(),
```
Comment (percona-ysorokin) [inline client/mysql.cc]: Use c++-style casts (`xxx_cast<char*>()`) for the new code. I believe `const_cast` is enough here.

File: client/mysql.cc [hunk 2]
```diff
@@ -3499,12 +3499,15 @@ com_help(String *buffer MY_ATTRIBUTE((unused)),
...
+           "\nFor information about Percona products and services, visit:\n"
            "   http://www.percona.com/\n"
-	   "Percona Server manual: http://www.percona.com/doc/percona-server/5.6\n"
+           "Percona Server manual: http://www.percona.com/doc/percona-server/%d.%d\n"
            "For the MySQL Reference Manual: http://dev.mysql.com/\n"
            "To buy Percona support, training, or other products, visit:\n"
-           "   https://www.percona.com/\n", INFO_INFO);
+           "   https://www.percona.com/\n",
+           MYSQL_VERSION_ID / 10000, (MYSQL_VERSION_ID / 100) % 100);
+  put_info((char*) glob_buffer.ptr(), INFO_INFO);
```
Comment (percona-ysorokin) [inline client/mysql.cc]: Unnecessary cast  - `glob_buffer.ptr()` is already `const char*`

**Review by percona-ysorokin — CHANGES_REQUESTED** [review]

**Comment by inikep** [comment]
Code is updated and tested at https://jenkins.percona.com/view/5.7/job/mysql-5.7-param/1392/

**Review by percona-ysorokin — APPROVED** [review]
Looks good to me (LGTM)

**Pull request merged**